Library/include/CSP/Systems/ECommerce/ECommerce.h

```diff
@@ -189,8 +226,38 @@ class CSP_API CheckoutInfoResult : public csp::services::ResultBase
 	CheckoutInfo CheckoutInformation;
 };
 
+
+/// @ingroup ECommerce System
+/// @brief Data class used to contain information when attempting to get Product Info.
```

Comment:
Is this comment right? Feels like it was copy/pasted from elsewhere?

Reply:
It was! 🤦 Will fix.